networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+// ExternalDNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= ExternalDNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The ExternalDNS Operator deploys and manages DNS external to the node.
+
+These release notes track the development of the ExternalDNS Operator in {product-title}.
+
+[id="external-dns-operator-release-notes-1.2.0"]
+== ExternalDNS Operator 1.2.0
+
+The following advisory is available for the ExternalDNS Operator version 1.2.0:
+
+* link:https://errata.devel.redhat.com/advisory/123003[RHEA-2022:5867 ExternalDNS Operator 1.0 operator/operand containers]
+
+== New features
+
+ExternalDNS now supports sharedVPC.
+
+For more information, see xref:../../networking/external_dns_operator/nw-creating-dns-records-on-aws.adoc#nw-control-dns-records-public-aws-with-VPC_creating-dns-records-on-aws[Creating DNS records on an public hosted zone for AWS by using Red Hat External DNS Operator].
+
+[id="external-dns-operator-1.2.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, ExternalDNS Operator issued an unnecessary warning during standard ExternalDNS operand pod deployments.
+
+*  link:https://bugzilla.redhat.com/show_bug.cgi?id=2086408[BZ#2086408]
+
+[id="external-dns-operator-release-notes-1.1.0"]
+== ExternalDNS Operator 1.1.0
+
+This release included a rebase of the operand from the upstream project version 0.13.1. The following advisory is available for the ExternalDNS Operator version 1.1.0:
```

Comment:
Here, we say "the ExternalDNS Operator," but oftentimes we don't have an article to accompany it. Should we use an article or not? probably should try and stay consistent :)